zip-1014.rst

```diff
@@ -0,0 +1,440 @@
+   Review Committee. The Major Grant Review Committee SHALL be selected by the
+   ZF's Community Panel. The Major Grant Review Committee's funding
+   decisions will be final, requiring no approval from the ZF Board, but are
+   subject to veto if the Foundation judges them to violate the ZF's operating
+   documents or U.S. law.
+
+7. Major Grant Review Committee members SHALL have a one-year term and MAY sit
+   for reelection. The Major Grant Review Committee is subject to the same
+   conflict of interest policy that governs the ZF board of directors
+   (i.e. they MUST recuse themselves when voting on proposals where they have
+   a financial interest). Additionally, no one with interest in or association
+   with the ECC may sit on the Major Grant Review Committee --- since the ECC
+   can be a beneficiary, this avoids those potential conflicts altogether.
+   The ZF SHALL continue to operate the Community Panel and SHOULD work
+   toward making it more representative and independent (more on that below).
+
+ZF SHALL recognize the ZF-MG slice of the Dev Fund as a Restricted Fund
+donation under the above constraints (suitably formalized), and keep separate
+accounting of its balance and usage under its Transparency and Accountability
+obligations defined below.
+
+From grant proposers' side, proposals for such grants SHALL be submitted
+through ZF's usual grant process, allowing for public discussion and public
+funding. It is intended that small one-time grants will be funded by drawing
+on the ZF-GU slice (where they also compete with other ZF activities), whereas
+large long-duration will be funded from the dedicated ZF-MG slice; though
+this is at ZF's discretion (e.g. if there are no Major Grant applications the
+ZF may opt to direct the ZF-MG to smaller grants).
+
+ZF SHALL strive to define target metrics and key performance indicators, and
+the Major Grant Review Committee SHOULD utilize these in its funding
+decisions.
+
+.. _mission: https://www.zfnd.org/about/#mission
+.. _values: https://www.zfnd.org/about/#values
+
+Direct-grant option
+'''''''''''''''''''
+
+It may be deemed better, operationally or legally, if the Major Grant funds
+are not accepted and disbursed by ZF, but rather directly assigned to the
+grantees. Thus, the following mechanism MAY be used in perpetuity, if agreed
+upon by both ECC and ZF before NU4 activation:
+
+Prior to each Network Upgrade, the Foundation SHALL publish a list of
+grantees' addresses and the total number of Dev Fund ZEC per block they
+should receive. ECC and ZF SHALL implement this list in any implementations
+of the Zcash consensus rules they maintain. This decision will then be,
+effectively, ratified by the miners as the network upgrade activates.
+
+
+Funding Target and Volatility Reserve
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Each Dev Fund slice has a Funding Target, initially US $700,000 for each
+slice. At the end of each calendar month, the fair market value of the Dev
```

Comment:
How will fair market value be calculated? The ZIP editors propose that each grant slice recipient should describe their methodology in the relevant transparency report.

Reply:
Addressed by https://github.com/ZcashFoundation/zips/pull/4